### PR TITLE
Bump OpenLayers and OL-Cesium versions

### PR DESCRIPTION
**Pull request by gberaudo**
With this change we are now based on stable versions.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -90,8 +90,8 @@
     "less-plugin-clean-css": "1.5.1",
     "ls": "0.2.1",
     "moment": "2.22.1",
-    "ol-cesium": "2.0.0",
-    "openlayers": "https://api.github.com/repos/openlayers/openlayers/tarball/f284b95cb897b3ce1b49b985b58ebe1ca76be1fe",
+    "ol": "5.1.3",
```
[the remark this answers is not in the document]
Comment (gberaudo) [reply package.json]: If I remember well it was the initial plan and I still think it is much better to rely on a stable branch than an obscure commit. That being said this PR may deserve more testing before merging.

[the remark this answers is not in the document]
Comment (sbrunner) [reply package.json]: But when OpenLayers 5.0 get out we reject this plan...

[the remark this answers is not in the document]
Comment (gberaudo) [reply package.json]: If you prefer I can change the PR base branch to be master. It is up to you.

[the remark this answers is not in the document]
Comment (sbrunner) [reply package.json]: +1 :-)

[the remark this answers is not in the document]
Comment (gberaudo) [reply package.json]: OK, done. Please review.

File: Makefile [hunk 2]
```diff
@@ -160,7 +160,7 @@ test: .build/node_modules.timestamp
 
 .PHONY: test-debug
 test-debug: .build/node_modules.timestamp .build/node_modules_karma-chrome-launcher.timestamp
-	./node_modules/karma/bin/karma start karma-conf.js --browsers=Chrome --single-run=false --autoWatch=true --debug
+	THEME=mobile ./node_modules/karma/bin/karma start karma-conf.js --browsers=Chrome --single-run=false --autoWatch=true --debug
```
Comment (gberaudo) [inline Makefile]: Ah, I guess this is no more required.

File: package.json [hunk 3]
```diff
@@ -91,8 +91,9 @@
     "moment": "2.22.2",
     "node-sass": "4.9.2",
     "node-sass-importer": "1.0.0",
-    "ol-cesium": "2.0.0",
-    "openlayers": "https://api.github.com/repos/openlayers/openlayers/tarball/f284b95cb897b3ce1b49b985b58ebe1ca76be1fe",
+    "moment": "2.22.1",
```
Comment (sbrunner) [inline package.json]: duplicated ...

**Pull request merged**